tubular/base.py

```diff
@@ -616,7 +630,7 @@ def __init__(
             getattr(df, pd_method_name)
 
         except Exception as err:
-            msg = f'{self.classname()}: error accessing "{pd_method_name}" method on pd.DataFrame object - pd_method_name should be a pd.DataFrame method'
+            msg = f'{self.classname()}: error accessing "{pd_method_name} "method on pd.DataFrame object - pd_method_name should be a pd.DataFrame method'  # noqa: E501
```

Comment:
sorry my fault this time haha: 

```suggestion
            msg = f'{self.classname()}: error accessing "{pd_method_name}" method on pd.DataFrame object - pd_method_name should be a pd.DataFrame method'  # noqa: E501
```